virttest/utils_conn.py

```diff
@@ -1688,6 +1691,8 @@ def conn_recover(self):
         (1).Delete remote file.
         (2).Restart libvirtd on server.
         """
+        traditional_mode = self.traditional_mode
```

Comment:
what it means 'traditional_mode' ? is it means mode for starting libvirtd ? monolith libvirtd vs modular libvirtd?
As far as I know, currently we still use  monolith libvirtd  to start libvirtd, so it should be default one.

Reply:
Yes, it is means monolith libvirtd.

Reply:
The meaning of this term or rather its scope would best be explained within the code for people that will not understand it. Just a remark here.